Scripts/DCS-BIOS/lib/modules/aircraft_modules/MiG-29A.lua

```diff
@@ -366,7 +366,20 @@ MiG_29A:definePotentiometer("MASTER_CAUTION_BRIGHTNESS_KNOB", devices.INTLIGHTS_
 
 -- HSI
 
--- Combined indicator (VVI/Turn/Slip indicator)
+-- ADI indicator
+local ADI = "Attitude director indicator"
```

Comment:
Honestly, I'd just go with ADI.
```suggestion
local ADI = "ADI"
```